exe/wallet/Main.hs

```diff
@@ -347,12 +344,9 @@ execHttpBridge args _ = do
     waitForConnection nw defaultRetryPolicy
     let tl = HttpBridge.newTransactionLayer @n
     wallet <- newWalletLayer @_ @(HttpBridge n) db nw tl
-    let settings = Warp.defaultSettings
-            & Warp.setPort walletPort
-            & Warp.setBeforeMainLoop (TIO.hPutStrLn stderr $
-                "Wallet backend server listening on: " <> toText walletPort
-            )
-    Server.start settings wallet
+    let logStartup port = TIO.hPutStrLn stderr $
+            "Wallet backend server listening on: " <> toText port
+    Server.start logStartup walletPort wallet
```

Comment:
I wonder if it wouldn't be slightly "cleaner" to let `start` takes a `Warp.Settings` as a parameter. But no strong feeling on that.